website/blog/2023-01-17-ferretdb-0-8-1-version-release.md

```diff
@@ -0,0 +1,77 @@
+tags: [release]
+---
+
+<!--truncate-->
+
+[FerretDB](https://www.ferretdb.io/) 0.8.1 is here, and it's better than ever!
+On the back of our Beta release, we've gone even further to improve FerretDB.
+And why shouldn’t we, especially when you consider the current economical landscape and the need for truly open source software?
+
+This release and the continued support of our amazing community furthers strengthens our drive to bring you the open source alternative to MongoDB.
+And now, as we approach our GA version later this quarter, we’re more emboldened to ensure FerretDB matches up with the behaviour you’d expect from MongoDB, and to create more compatibility with other applications and real world use cases.
+
+Keep reading to learn more.
+
+## New features
+
+We'll start off with probably the most anticipated update!
+It is surely a concern having to use an older version of FerretDB, without knowing if a new version release is available or not.
+
+So first up on our list of updates for this release, we are delighted to announce that we now report the availability of newer versions of FerretDB in mongosh.
+However, this is only available for users with telemetry enabled.
+If telemetry is enabled and a newer version of FerretDB is available, you'll be notified and can stay up-to-date with the latest and best version of FerretDB.
+
+If you are yet to enable telemetry and you’d love to access and enable this feature, please [see our documentation here](https://a9b5c3ea.ferretdb-docs-dev.pages.dev/telemetry/).
+
+Next, we've implemented the `distinct` command in FerretDB.
+With this command, you can easily find the unique values of a specific key in your data.
+The `distinct` command takes three arguments: `distinct`, `key`, and `query`.
+For instance, if you want to find the unique values of the "age" field in a specific "people" collection, run the command below:
+
+```js
+db.runCommand( { distinct: "people", key: "age", query: {} } )
+```
+
+The `distinct` argument essentially specifies the collection you want to query, the key argument specifies the field you want to find unique values for, and the query argument allows you to filter the results.
+
+Even better, FerretDB now supports the `$rename` field update operator, enabling you to rename fields in a document without changing its contents.
+You can use the operator this way:
+
+```js
+db.collection.update( { }, { $rename: { "oldField": "newField" } }, { multi: true } )
+```
+
+Additionally, we are continuously improving our authentication and security process.
+And for that reason, we've also included a way to validate a client's TLS certificate when the root CA certificate is provided.
+In essence, this makes it possible to configure FerretDB to validate a client's certificates against the given CA certificate and reject connections without valid certificates.
+
+## Bug fix
+
+With this release, we've also fixed a bug with the `distinct` command where filter wasn't applied to it.
```

Comment:
I think this sounds weird, since distinct a new feature added in this release. I think this is not needed.